publisher-sdk/src/main/java/com/criteo/publisher/BidListener.kt

```diff
@@ -16,14 +16,15 @@
 
 package com.criteo.publisher
 
+import androidx.annotation.NonNull
 import com.criteo.publisher.model.CdbResponseSlot
 
 /**
  * Callback through which bid responses will be served asynchronously.
  * Each of the callback methods will only be called once.
  */
 interface BidListener {
-  fun onBidResponse(cdbResponseSlot: CdbResponseSlot)
+  fun onBidResponse(@NonNull cdbResponseSlot: CdbResponseSlot)
```

Comment:
What's the value of `@NonNull` annotation in Kotlin code?

Reply:
![image](https://user-images.githubusercontent.com/3816099/94176832-f4934e00-fe98-11ea-8db2-b3567631ef57.png)

You're right that's a mistake. My implementation automatically added a `@NonNull` and I was confused why the interface didn't had one. And I didn't get that it was Kotlin code, not Java.